src/guides/v2.3/ui_comp_guide/concepts/magento-bindings.md

```diff
@@ -8,13 +8,14 @@ title: Magento binding syntax
 
 Within HTML templates, Magento gives you the option of using a binding syntax that is simpler and easier to read and write than the standard Knockout binding syntax. The following code snippets help make the comparison.
 
-### Knockout syntax
+### Knockout native syntax
 
 ```html
-<!-- ko if: isVisible-->
+<!-- ko if: isVisible -->
     <div class="someClass">
-        <!-- ko i18n: 'Some translatable message!'--><!-- /ko -->
+        <!-- ko i18n: 'Some translatable message!' --><!-- /ko -->
         <span data-bind="html: content"></span>
+        <span data-bind="i18n: 'translatable text'"></span>
```

Comment:
Seems like this part should be also added in magento version

Reply:
i have reverted back to origin for maintain backward compatible